ci-operator/step-registry/openshift-extended/netobserv-test/openshift-extended-netobserv-test-ref.yaml

```diff
@@ -0,0 +1,58 @@
+ref:
+  as: openshift-extended-netobserv-test
+  from: tests-private
+  grace_period: 10m
+  commands: openshift-extended-netobserv-test-commands.sh
+  cli: latest
+  timeout: 12h0m0s
+  env:
+  - name: TEST_SCENARIOS
+    default: |
+        Network_Observability
```

Comment:
@kuiwang02 @jianzhangbjz Is it expected that subteams copy the `openshift-extended-test` step and overwrite the envs ?

Reply:
updated